dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/renderers/html/htmlFormatingUtils.kt

```diff
@@ -23,14 +23,17 @@ public fun FlowContent.buildTextBreakableAfterCapitalLetters(name: String, hasLa
     }
 }
 
-public fun FlowContent.buildBreakableDotSeparatedHtml(name: String) {
-    val phrases = name.split(".")
+/**
+ * Makes [name] breakable after each occurrence of [breakableChar].
```

Comment:
```suggestion
 * Makes [name] breakable by inserting `<wbr>` element after each occurrence of [breakableChar].
```

Reply:
Done in [2ca4577](https://github.com/Kotlin/dokka/pull/3240/commits/2ca4577c0e2e488e621b87979218f634595ac356) :+1: